hangman.js

```diff
@@ -1 +1,107 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word = prompt('gimme a word:');
+var guesses = [];
+var attempts = word.length + 4;
+var attemptsLeft = attempts;
+var display = '';
+
+myl = 'a';
```

Comment:
I'm not sure if you intended to leave this global variable here, but it doesn't appear your using it in your hangman game